CNCF suggests adding benefits/value proposition section to introduction (#530)
kubewarden/docs: docs/introduction.md

Kubewarden is a Kubernetes Policy Engine.
It aims to be the Universal Policy Engine for Kubernetes.

Kubewarden is a [CNCF](https://cncf.io) Sandbox project,
originally created by [SUSE Rancher](https://www.rancher.com/).

## How does Kubewarden help?

Kubewarden offers flexibility for policy admission and enforcement in a Kubernetes environment.

### Benefits and value

- Use any programming language that generates [WebAssembly](https://webassembly.org) binaries to write your policies.

> - Utilizing WebAssembly allows policy compatibility across processors and operating systems.

Reviewer: ```suggestion
- WebAssembly enables policy compatibility across processors and operating systems.
```

Reviewer: Here I suggested rephrasing to match the other items in the list that do not use a gerund.

> - Reuse of policies from other from other policy engines without the need for rewriting them.

Reviewer: ```suggestion
- Reuse of policies from other policy engines without the need for rewriting them.
```

- Distribute policies using standard and secure mechanisms such as [OCI](https://opencontainers.org) compliant registries.
- Policy enforcement at admission ensures only compliant workloads run.
- The Kubewarden audit scanner actively and continuously checks policy enforcement over time.
- Verify policies using [SLSA](https://slsa.dev) (Supply Chain Levels for Software Artifacts) tools and practices.
- Kubewarden provides a comprehensive approach to admission policy management.

> - CNCF membership and a growing open source community, and ecosystem, supports Kubewarden helping with transparency, collaboration, and improvement.

Reviewer: ```suggestion
- CNCF membership and a growing open source community and ecosystem help Kubewarden with transparency, collaboration and improvement.
```
